fast64_internal/oot/data/oot_actor_data.py

```diff
@@ -0,0 +1,40 @@
+from dataclasses import dataclass
+from .oot_getters import getRoot, getEnumList
+from os import path
+
+
+@dataclass
+class OoT_Actor:
+    id: str
+    key: str
+    name: str
+    category: str
+    tiedObjects: str
+
+
+class OoT_ActorData:
+    """Everything related to OoT Actors"""
+
+    def __init__(self):
+        # Path to the ``ActorList.xml`` file
+        self.actorXML: str
```

Comment:
I think this can be a normal variable, doesn't have to be a class member?
```suggestion
        actorXML: str
```
also this line is not really needed anyway with `actorXML` being set below